test/evmc/mocked_host.hpp

```diff
@@ -313,5 +321,59 @@ class MockedHost : public Host
     {
         recorded_logs.push_back({addr, {data, data_size}, {topics, topics + topics_count}});
     }
+
+    /// Record an account access.
+    ///
+    /// This method is required by EIP-2929 introduced in ::EVMC_BERLIN. It will record the account
+    /// access in MockedHost::recorded_account_accesses and return previous access status.
+    /// This methods returns ::EVMC_ACCESS_WARM for known addresses of precompiles.
+    /// The EIP-2929 specifies that evmc_message::sender and evmc_message::destination are always
+    /// ::EVMC_ACCESS_WARM. Therefore, you should init the MockedHost with:
+    ///
+    ///     mocked_host.access_account(msg.sender);
+    ///     mocked_host.access_account(msg.destination);
+    ///
+    /// The same way you can mock transaction access list (EIP-2930) for account addresses.
+    ///
+    /// @param addr  The address of the accessed account.
+    /// @returns     The ::EVMC_ACCESS_WARM if the account has been accessed before,
+    ///              the ::EVMC_ACCESS_COLD otherwise.
+    evmc_access_status access_account(const address& addr) noexcept override
```

Comment:
```suggestion
    evmc_access_status access_account(address const& addr) noexcept override
```
Don't we put the `const`-modifier on the right hand side? (also on the other occurrences)

Reply:
We are not modifying imported sources.